sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredentialBuilder.java

```diff
@@ -186,12 +186,13 @@ public DefaultAzureCredential build() {
     }
 
     private ArrayList<TokenCredential> getCredentialsChain() {
-        ArrayList<TokenCredential> output = new ArrayList<TokenCredential>(6);
+        ArrayList<TokenCredential> output = new ArrayList<TokenCredential>(7);
         output.add(new EnvironmentCredential(identityClientOptions.clone()));
         output.add(new ManagedIdentityCredential(managedIdentityClientId, managedIdentityResourceId, identityClientOptions.clone()));
         output.add(new SharedTokenCacheCredential(null, IdentityConstants.DEVELOPER_SINGLE_SIGN_ON_ID,
             tenantId, identityClientOptions.clone()));
         output.add(new IntelliJCredential(tenantId, identityClientOptions.clone()));
+        output.add(new AzureDeveloperCliCredential(tenantId, identityClientOptions.clone()));
```

Comment:
For now we are putting this credential right after `ManagedIdentityCredential`. We may change this after archboard review. In the meantime, we should be consistent with other languages.

Reply:
updated